install/tools/istio_vm_common.sh

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
+
+# Helper functions for extending the cluster with external VMs.
+
+# Script can be sourced in other files or used from tools like ansible.
+# Environment variables used:
+
+# ISTIO_NAMESPACE - if not set the default in .kube/config is used.
+# ISTIO_FILES - directory where istio artifacts are downloaded, default to current dir
+
+# Initialize internal load balancers to access K8S DNS and Istio Pilot, Mixer, CA.
+# Must be run once per cluster.
+function istioInitILB() {
+   local NS=""
+   if [[ ${ISTIO_NAMESPACE:-} != "" ]]; then
+     NS="-n $ISTIO_NAMESPACE"
+   fi
+
+cat <<EOF | kubectl apply $NS -f -
+apiVersion: v1
+kind: Service
+metadata:
+  name: istio-pilot-ilb
+  annotations:
```

Comment:
Istio-pilot-ilb-gcp

Reply:
There are a bunch of other clusters using ILB, I didn't add the other annotations since I 
didn't have time to test them. I think in the case of bluemix ( and possibly 
others ) this creates a regular internal service (annotation ignored), but still routable from the VMs.

Maybe I should call it istio-pilot-ext ? ( since in some cases an ILB is not needed )

Reply:
It's not gcp-specific, the service will be used for other providers as well.
(and we'll add annotations as needed by the other providers )

Reply:
istio-pilot-alt?

Reply:
or leave it as is

Reply:
I'll leave it as is if you don't mind, we can probably merge it into the main configs ( just add annotations to the 'real service' ) after we get more testing.